app/main.py

```diff
@@ -1,3 +1,58 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: float) -> Distance:
```

Comment:
`other `argument could be `int` or `Distance` class instance as well, improve your type annotation to encounter that where applicable.
```suggestion
    def __add__(self, other: float | int | Distance) -> Distance:
```